siglens/results/c6a.4xlarge.json

```diff
@@ -1,55 +1,54 @@
     "tags": ["Go", "logs", "search", "SigLens", "observability"],
-    "load_time": 5198.83,
-    "data_size": 28396387,
+    "load_time": 5070.66,
+    "data_size": 28572908,
     "result": [
-        [0.209, 0.082, 0.082],
-        [0.602, 0.308, 0.306],
-        [0.162, 0.082, 0.079],
-        [0.182, 0.074, 0.080],
-        [0.172, 0.081, 0.086],
-        [0.155, 0.075, 0.086],
-        [0.168, 0.083, 0.080],
-        [0.625, 0.886, 0.178],
-        [5.415, 4.735, 4.679],
-        [7.755, 6.464, 6.525],
-        [2.299, 1.369, 0.639],
-        [9.720, 0.687, 0.683],
-        [0.318, 0.223, 0.243],
-        [0.407, 0.264, 0.259],
-        [0.372, 0.243, 0.240],
-        [0.262, 0.181, 0.181],
-        [0.350, 0.233, 0.229],
-        [0.349, 0.218, 0.218],
-        [null, null, null],
-        [1.020, 1.066, 0.256],
-        [10.224, 2.242, 2.237],
-        [13.308, 2.421, 1.683],
-        [23.581, 3.912, 2.874],
-        [14.173, 1.890, 1.362],
-        [12.512, 8.067, 8.055],
-        [10.056, 7.677, 7.659],
-        [12.969, 10.032, 10.020],
-        [12.546, 6.239, 5.523],
-        [131.366, 129.492, 128.019],
-        [0.177, 0.095, 0.094],
-        [0.464, 0.283, 0.273],
-        [0.651, 0.355, 0.325],
-        [0.673, 0.475, 0.448],
-        [0.525, 0.328, 0.333],
-        [0.536, 0.344, 0.317],
-        [0.261, 0.183, 0.189],
-        [12.303, 6.702, 6.646],
-        [11.801, 6.439, 6.456],
-        [2.272, 1.155, 1.140],
-        [7.274, 5.705, 5.726],
-        [2.006, 1.113, 1.106],
-        [4.910, 2.525, 2.540],
-        [10.214, 7.921, 7.923]        
-    ]
+        [0.236, 0.082, 0.072],
+        [1.430, 0.202, 0.193],
+        [0.169, 0.077, 0.081],
+        [0.178, 0.081, 0.084],
+        [0.179, 0.077, 0.081],
+        [0.175, 0.079, 0.083],
+        [0.185, 0.086, 0.082],
+        [0.399, 0.858, 0.144],
+        [3.749, 3.427, 3.421],
+        [5.306, 4.735, 4.741],
+        [2.250, 1.210, 0.501],
+        [9.135, 0.540, 0.507],
+        [0.258, 0.188, 0.182],
+        [0.306, 0.217, 0.219],
+        [0.291, 0.204, 0.203],
+        [0.205, 0.139, 0.139],
+        [0.259, 0.169, 0.179],
+        [0.265, 0.174, 0.172],
+        [null, null, null, ],
+        [0.784, 1.022, 0.225],
+        [5.167, 1.620, 1.612],
+        [6.201, 1.972, 1.304],
+        [11.386, 2.936, 2.234],
+        [6.570, 1.676, 1.032],
+        [4.879, 1.500, 1.502],
+        [3.299, 1.434, 1.421],
+        [3.926, 1.889, 1.880],
+        [5.901, 4.415, 3.738],
+        [13.353, 12.047, 12.046],
+        [0.188, 0.092, 0.091],
+        [0.334, 0.211, 0.206],
+        [0.408, 0.273, 0.270],
+        [0.433, 0.301, 0.295],
+        [0.356, 0.251, 0.246],
+        [0.364, 0.260, 0.255],
+        [0.217, 0.138, 0.138],
+        [7.261, 4.585, 4.451],
+        [7.374, 4.394, 4.311],
+        [1.658, 0.960, 0.975],
+        [4.143, 3.108, 3.228],
+        [1.490, 0.831, 0.800],
+        [3.355, 1.998, 1.977],
+        [5.071, 4.144, 4.168],    

```

Comment:
The trailing comma destroyed the UI. Fix: https://github.com/ClickHouse/ClickBench/pull/425